tests/TailwindBinaryTest.php

```diff
@@ -34,13 +35,11 @@ public function testBinaryIsDownloadedAndProcessCreated()
 
         $binary = new TailwindBinary($binaryDownloadDir, __DIR__, null, 'fake-version', $cache, null, $client);
         $process = $binary->createProcess(['-i', 'fake.css']);
-        $this->assertFileExists($binaryDownloadDir.'/fake-version/'.TailwindBinary::getBinaryName());
-
-        // Windows doesn't wrap arguments in quotes
-        $expectedTemplate = '\\' === \DIRECTORY_SEPARATOR ? '"%s" -i fake.css' : "'%s' '-i' 'fake.css'";
+        $binaryFile = $binaryDownloadDir.'/fake-version/'.TailwindBinary::getBinaryName();
+        $this->assertFileExists($binaryFile);
 
         $this->assertSame(
-            \sprintf($expectedTemplate, $binaryDownloadDir.'/fake-version/'.TailwindBinary::getBinaryName()),
+            (new Process([$binaryFile, '-i', 'fake.css'], __DIR__))->getCommandLine(),
```

Comment:
A comment here describing why you're asserting the command line that way would be handy